server/apiserver/apiserver.go

```diff
@@ -50,7 +49,7 @@ type Client struct {
 	Pool *Pool
 }
 
-var cooldown = 300000
+var cooldown, _ = time.ParseDuration("5m")
```

Comment:
nit: i think that i'd prefer something like this rather than the ParseDuration call:

```go
// i think these can be const refs
const (
   fiveMinutes = 5 * 60 * 1000000000
   cooldown = time.Duration(fiveMinutes)
)
```